Add `sd-webui-ar-plus` extension

LEv145 opened the pull request:
Fork of `Aspect Ratio selector` with new features and logic for aspect ratio

Fork features: https://github.com/LEv145/--sd-webui-ar-plus#fork-features

w-e-w commented:
I assume that you're not planning to push your changes back up to the base repo?

LEv145 commented:
I'm afraid I've changed the logic and repository too much to migrate all the changes from there
I completely changed the logic of the `aspect ratio` and removed some buttons

w-e-w commented:
I was concerned about if you're going to merge the change to upstream then fork will become deprecated
so I can consider it as a separate extension
